prod-aws/kafka-shared-msk/customer-support/__env.tf

```diff
@@ -0,0 +1,20 @@
+terraform {
```

Comment:
Can you please symlink this to https://github.com/utilitywarehouse/kafka-cluster-config/blob/main/prod-aws/kafka-shared-msk/__env.tf?

Reply:
ah knew I'd missed something

Reply:
Done